src/index.html

```diff
@@ -8,7 +9,59 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="hearder">
+      <div class="header__container">
+
+        <div class="header__logo">
+          <a href="#">
+            <img src="./images/logo.png" alt="Moyo">
+          </a>
+        </div>
+
+        <nav class="header__menu menu">
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link is-active">Apple</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Samsung</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#" class="menu__link">Smartphones</a>
+            </li>
+
+            <li class="menu__item">
+              <a
```

Comment:
```suggestion
              <a
                data-qa="hover"
                href="#"
                class="menu__link"
              >
                Laptops & Computers
              </a>
```

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3866/files#r1331654923 
didn't fix it